news/2022-05-20-beatmap-spotlights-season-6-spring-2022-high-speed.md

```diff
@@ -0,0 +1,283 @@
+- [Kaneko Chiharu - iLLness LiLin (tasuke912) \[TASUKE'S EXHAUST\]](https://osu.ppy.sh/beatmapsets/777109#fruits/1632661), suggested by x_angelkawaii_x
+- [nao - Towa naru Kizuna to Omoi no Kiseki (rew0825) \[Miracle\]](https://osu.ppy.sh/beatmapsets/601392#fruits/1566770), suggested by x_angelkawaii_x
+- [Doma Umaru \[CV.Tanaka Aimi\] - Kakushinteki\*Metamaruphose! (CLSW) \[CrystalSeed's UmaruDosE!!\]](https://osu.ppy.sh/beatmapsets/332623#fruits/773257), suggested by x_angelkawaii_x
+- [Horie Yui - The World's End (Bastian and Greaper) \[Bastian & Greaper's Finale\]](https://osu.ppy.sh/beatmapsets/1134081#fruits/2368389), suggested by wonjae
+- [Rish feat. Choko - Punai Punai Taisou (Kukkai) \[271 271\]](https://osu.ppy.sh/beatmapsets/1560368#fruits/3186972), suggested by x_angelkawaii_x
+
+### osu!mania (led by [Crumpey](https://osu.ppy.sh/users/3518705))
+
+#### Playlist A (high speed)
+
+- [DJ TOTTO - Crystalia (Tofu1222) \[CS' Hyper\]](https://osu.ppy.sh/beatmapsets/782749#mania/1643587), suggested by Crumpey
+- [katagiri - HIVEMIND (DannyPX) \[Hyper\]](https://osu.ppy.sh/beatmapsets/1665442#mania/3411830), suggested by Crumpey
+- [Umeboshi Chazuke - Bison Charge (\_Asha) \[Insane\]](https://osu.ppy.sh/beatmapsets/838465#mania/1785223), suggested by Crumpey
+- [Five Hammer - fffff op.2 (\_Stan) \[bie's Insane\]](https://osu.ppy.sh/beatmapsets/1452883#mania/2987140), suggested by lenpai
+- [Frums - Star of the COME ON!! (Kawawa) \[INSANE!!\]](https://osu.ppy.sh/beatmapsets/1305627#mania/2706983), suggested by lenpai
+- [Camellia - \* Erm, could it be a Spatiotemporal ShockWAVE Syndrome...? (Dubstek) \[YABAI\]](https://osu.ppy.sh/beatmapsets/1477789#mania/3031881), suggested by Crumpey
+- [Hypernite Industries - Speedcore 300 (nowsmart) \[Extra\]](https://osu.ppy.sh/beatmapsets/228853#mania/532796), suggested by Crumpey
+- [Imperial Circus Dead Decadence - Shinbatsu o Tadori Kyoukotsu ni Itaru (Evening) \[Till November Night (reprise)\]](https://osu.ppy.sh/beatmapsets/1647221#mania/3362200), suggested by Crumpey
+- [Frums - VIS::CRACKED (\[Crz\]Crysarlene) \[Mipha vs. Crys's Oblivion\]](https://osu.ppy.sh/beatmapsets/1561114#mania/3188294), suggested by Crumpey
+
+Playlist B and C will be available on the [Beatmap Spotlights article](/wiki/Beatmap_Spotlights/Seasons/2022_Spring) as we open the playlist lobbies, so stay tuned!
+
+## The Future
+
+The new Beatmap Spotlights is still a project in its early development stage and depends a lot on community feedback and engagement to improve. Many things can (and will) change for future seasons, and it all depends on *your* feedback!
+
+What do you want for the next season? What needs to be fixed? What are your opinions on the current system? You can voice out all your opinions [here](https://osu.ppy.sh/community/forums/topics/1579425) and we will keep note of every suggestion, so please let us know how you feel about things.
```

Comment:
the post is inaccessible

Reply:
yeah, it's invisible atm, will be moved to the public view when its about to get merged